js/ui/scheduler/ui.scheduler.subscribes.js

```diff
@@ -52,11 +52,14 @@ const subscribes = {
             max: dateRange[1]
         };
 
-        let dates = recurrenceUtils.getDatesByRecurrence(recurrenceOptions);
+        let dates = recurrenceUtils.getDatesByRecurrence(recurrenceOptions),
+            initialDates;
```

Comment:
Maybe, should  initialDates = dates be here?

Reply:
It is an unnecessary operation, because in case of empty "dates" array, operation "=" will be executed twice.